### ISSUE TITLE
[Design] Create a simple logo

### ISSUE THREAD

**Issue by markusphil** [OpenTechSchool-Leipzig/poll_app#40]

**Comment by AlexanderBoenninger** [comment]
Das würde ich am Ende machen, wenn die App steht.

**Comment by markusphil** [comment]
@AlexanderBoenninger you're right: this task has a low priority right now. I just wanted to provide some more tasks for people that can't or don't want to work on the code base 😊
